alsongDalsong/alsongDalsong/alsongDalsong/Sources/Components/UIKitComponents/ASAlertController.swift

```diff
@@ -90,14 +125,41 @@ class ASAlertController: UIViewController {
 }
 
 extension ASAlertController {
-    convenience init(titleText: String, doneButtonTitle: String, cancelButtonTitle: String, textFieldPlaceholder: String) {
+    convenience init(titleText: String, doneButtonTitle: String, cancelButtonTitle: String, textFieldPlaceholder: String, isUppercased: Bool = false, style: ASAlertStyle) {
         self.init()
         self.titleText = titleText
         self.textFieldPlaceholder = textFieldPlaceholder
         self.doneButtonTitle = doneButtonTitle
         self.cancelButtonTitle = cancelButtonTitle
+        self.isUppercased = isUppercased
+        self.style = style
         
         self.modalTransitionStyle = .crossDissolve
         self.modalPresentationStyle = .overFullScreen
     }
 }
+
+extension ASAlertController: UITextFieldDelegate {
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        let uppercaseString = string.uppercased()
+        
+        if let text = textField.text,
+           let textRange = Range(range, in: text)
+        {
+            let updatedText = text.replacingCharacters(in: textRange, with: uppercaseString)
+            if updatedText.count <= textMaxCount {
+                textField.text = updatedText
+                return false
+            } else {
+                return false
+            }
```

Comment:
```suggestion
            if updatedText.count <= textMaxCount {
                textField.text = updatedText
            } 
            return false
```
둘 다 false를 리턴한다면 하나로 줄일 수 있을 것 같아요